examples/example.py

```diff
@@ -21,7 +23,7 @@ def main():
     tester = False

```

Comment:
what is this?

Reply:
i used to use it to test for valid input from user

Reply:
So I think it needs a better name ;)